.github/buildomat/jobs/opte-api.sh

```diff
@@ -3,7 +3,7 @@
 #: name = "opte-api"
 #: variety = "basic"
 #: target = "helios-2.0"
-#: rust_toolchain = "nightly-2024-02-06"
+#: rust_toolchain = "nightly-2024-04-25"
```

Comment:
Dumb question, but why doesn't `rust-toolchain.toml` work here?

Reply:
I believe this is down to us needing several nightly format options from `cargo fmt` across the repo. However, everything apart from XDE compiles on a stable toolchain version (as XDE is reliant on several unstable feature flags).